main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+
+	"github.com/talos-systems/os-engine/internal/controller"
+	"github.com/talos-systems/os-engine/internal/loader"
+	"github.com/talos-systems/os-engine/pkg/events"
+	"github.com/talos-systems/os-engine/pkg/sink"
+)
+
+type Options struct {
+	Runtime string
+	Plugins string
+	Socket  string
+	Debug   bool
+}
+
+var options Options
+
+func init() {
+	flag.StringVar(&options.Runtime, "runtime", "/lib/engine/runtime/runtime.so", "the path to the runtime")
```

Comment:
What would this be?
pkg/runtime points to this being an interface that implements a sink but still not super clear on what `/lib/engine/runtime/runtime.so` could be.

Reply:
:thinking: Still working on how engine and runtime work together (and if they even should be separate). I think I can address this in the "concepts" doc.